test/fixtures/user/user.js

```diff
@@ -253,5 +253,29 @@ module.exports = () => {
         url: "https://res.cloudinary.com/realdevsquad/image/upload/v1667685133/profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar.jpg",
       },
     },
+    {
+      username: "ritvik",
+      github_id: "RitvikJamwal75",
+      first_name: "Ritvik",
+      yoe: 1,
+      picture: {
+        publicId: "profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar",
+        url: "https://res.cloudinary.com/realdevsquad/image/upload/v1667685133/profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar.jpg",
+      },
+      incompleteUserDetails: false,
+      status: "active",
+      last_name: "Jamwal",
+      github_display_name: "Ritvik Jamwal",
+      website: "RitvikJamwal75.github.io/portfolio",
+      designation: "Sw Engineer",
+      company: "Goldman Sacks",
+      instagram_id: "ritvikjmwal",
+      roles: {
+        archived: true,
+        member: false,
+      },
+      twitter_id: "RitvikJamwal4u",
```

Comment:
Please use dummy data rather than using the actual data
also, we have used a real company name 
companies can ask for royalty
Please change the complete data to some dummy names and URLs.

Reply:
Done...please check latest commit